azureeyemodule/app/model/ssd.cpp

```diff
@@ -129,6 +177,60 @@ cv::GStreamingCompiled SSDModel::compile_cv_graph() const
     return pipeline;
 }
 
+cv::GStreamingCompiled SSDModel::compile_cv_graph_uvc_video() const
```

Comment:
Again, this is a ton of code duplication. It would be better if we could combine this method with the other `compile_graph` method, or at the very least, use the one in the super class.

The only reason for this method to be different from the one in the super class (that I can tell) is that you are using the G-API to do the inference rendering (bounding box drawing). That's another reason in favor of not doing it that way, and instead just drawing the bounding boxes like we've been doing it, using the traditional OpenCV library calls.

Reply:
as answered above.